libraries/AP_DDS/AP_DDS_Topic_Table.h

```diff
@@ -382,4 +385,22 @@ constexpr struct AP_DDS_Client::Topic_table AP_DDS_Client::topics[] = {
         },
     },
 #endif // AP_DDS_GLOBAL_POS_CTRL_ENABLED
+#if AP_DDS_MOT_CTRL_ENABLED
+    {
+        .topic_id = to_underlying(TopicIndex::MOTOR_CONTROL_SUB),
+        .pub_id = to_underlying(TopicIndex::MOTOR_CONTROL_SUB),
+        .sub_id = to_underlying(TopicIndex::MOTOR_CONTROL_SUB),
+        .dw_id = uxrObjectId{.id=to_underlying(TopicIndex::MOTOR_CONTROL_SUB), .type=UXR_DATAWRITER_ID},
+        .dr_id = uxrObjectId{.id=to_underlying(TopicIndex::MOTOR_CONTROL_SUB), .type=UXR_DATAREADER_ID},
+        .topic_rw = Topic_rw::DataReader,
+        .topic_name = "rt/ap/cmd_mot",
```

Comment:
Should we make this an experimental topic till we've flown it on a vehicle?

Reply:
Got it